### PR TITLE
Added hagelslag recipe

### PR DESCRIPTION
**Pull request by djgagne**
Checklist

- [ x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ x] Source is from official source
- [ x] Package does not vend other packages
- [ x] Build number is 0
- [ x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by djgagne** [comment]
I am willing to be a maintainer for this project.

File: recipes/hagelslag/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  number: 0
+  skip: True  # [win]
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
```
Comment (marcelotrevisani) [inline recipes/hagelslag/meta.yaml]: Could you please remove all the comments?

File: recipes/hagelslag/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
```
Comment (marcelotrevisani) [inline recipes/hagelslag/meta.yaml]: I think this project is a pure python project right?
If so, you do not need the compiler

**Comment by djgagne** [comment]
I have removed the comments and the C-compiler line.

File: recipes/hagelslag/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/h/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 44c4add4c37acd116a4bb7a4b4d39b3d68f88818f3d6442b3f4093e68c1811d6
+
+build:
+  number: 0
+  skip: True
```
Comment (xhochy) [inline recipes/hagelslag/meta.yaml:14]: This just skips the build of the package completly.

File: recipes/hagelslag/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    - pandas >=0.19
+    - netcdf4
+    - pygrib
+    - basemap
+    - arrow
+    - scikit-learn >=0.2
+    - scikit-image
+    - scipy >=1.0
+    - pyproj
+    - matplotlib >=2.0
```
Comment (xhochy) [inline recipes/hagelslag/meta.yaml:33]: Please use `matplotlib-base`

**Pull request closed without merging**